transcom/mymove
# B-20303 - update prime api with emplid

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A973564&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## INT PR
https://github.com/transcom/mymove/pull/13188

## Summary

Update the Prime API to include EMPLID when present.

### How to test

1. Create a Coast Guard member with EMPLID.
2. Create a HHG and move it through TOO approval so that it shows up in Prime Simulator.
3. Setup Postman to run the move-task-orders API call.
    /prime/v3/move-task-orders/{moveID}
4. Verify that you see the new emplid variable under the order/customer/ in the result set.
5. Create a non Coast Guard member, run through steps 2 and 3, then verify that you do not see a emplid variable in the result set.
